apps.json

```diff
@@ -133,5 +133,10 @@
   },
   "guc-desktop": {
     "name": "GUC"
+  },
+  "dpoll": {
+    "name": "Dpoll",
+    "homepage": "https://dpoll.xyz",
+    "url_scheme": "https://dpoll.xyz/detail/@{username}/{permlink}"
```

Comment:
It seem on dpoll that the article url have an extra `/` in the end like `https://dpoll.xyz/detail/@revan746/hangi-borsalar-daha-aktiv-kullanyorsunuz-which-exchanges-do-you-use-more/`